test/new-e2e/utils/e2e/client/agent_cmd_runner.go

```diff
@@ -36,7 +35,7 @@ func newAgentCommandRunner(t *testing.T, executeAgentCmdWithError executeAgentCm
 	return agent
 }
 
-func (agent *AgentCommandRunner) executeCommand(command string, commandArgs ...AgentArgsOption) string {
+func (agent *AgentCommandRunner) executeCommand(command string, commandArgs ...AgentArgsOption) (string, error) {
```

Comment:
some of the executed cmds return 255 and errors that we want under test, so I updated this func signature

Reply:
I reverted this to avoid conflicts https://github.com/DataDog/datadog-agent/pull/18771/commits/9d0ce6abae3db2c2cc1e788d8edc5579318f0b14